bin/run_tests.sh

```diff
@@ -0,0 +1,10 @@
+# This script is used to run tests in Meson
+script_path=`realpath "$0"`
+bin_dir=`dirname "$script_path"`
+base_dir=`dirname "$bin_dir"`
+
+export SHELL="$base_dir/build/src/cmd/ksh93/ksh"
+cd "$base_dir/src/cmd/ksh93/tests"
+
+# Skip shcomp tests for now
+"$base_dir/build/src/cmd/ksh93/ksh" ./shtests --posix --utf8
```

Comment:
I would use the `SHELL` var you exported above to help ensure the paths don't get out of sync.